renku/core/commands/init.py

```diff
@@ -353,7 +353,7 @@ def fetch_template_from_git(source, ref="master", tempdir=None):
 
     try:
         # fetch ref and set the HEAD
-        template_repo.remotes.origin.fetch(ref)
+        template_repo.remotes.origin.fetch()
```

Comment:
@rokroskar @Panaetius note this only adds support for commit sha. support for tag would still fail. personally having a support for checking out a tag would be nice. but i'm wondering whether you think that'd would be as well desirable.

Reply:
actually based on the docstring this function advertises that
```
    :param ref: reference for git checkout - branch, commit or tag
```
so i'll fix it for tag as well.